### PR TITLE
Fix typos in documentation

### PR DESCRIPTION
**Pull request by jonovs**
I ran the docs through a spell checker to fix the typos and punctuation issues. I also improved sentence structure and word usage in several places.

In the case of spelling, there seems to be a mix of American and British English, e.g. `standardise` (Br.E.) and `authorized` (Am.E.). Is there a variant you prefer more, @ivarconr ?

### CODE REVIEW

**Comment by ivarconr** [comment]
Thanks :+1: 

> I ran the docs through a spell checker to fix the typos and punctuation issues

What tools do you use for that? I have tried to have some spellchecking as part of Visual Studio Code, but does not seem to handle much more than basic word checking. 

> In the case of spelling, there seems to be a mix of American and British English, e.g. standardise (Br.E.) and authorized (Am.E.). Is there a variant you prefer more, @ivarconr ?

Personally I prefer British English

File: docs/activation-strategies.md [hunk 1]
```diff
@@ -45,23 +45,23 @@ Gradually activates feature toggle. Stickiness based on session id. It is almost
 
 ## gradualRolloutRandom
 
-Randomly activates the feature toggle. No stickiness. We have found this rollout strategy very useful in some scenarios, especially when we enable a feature which is not visible to the user. It is also the strategy we use to sample metrics and error reports.
+This gradual rollout random strategy randomly activates a feature toggle and has no stickiness. We have found this rollout strategy very useful in some scenarios, especially when we enable a feature which is not visible to the user. It is also the strategy we use to sample metrics and error reports.
```
Comment (ivarconr) [inline docs/activation-strategies.md]: gradual rollout random strategy => `GradualRolloutRandom` strategy in order to be consistent?

Comment (jonovs) [reply docs/activation-strategies.md]: Yup, consistency is good!

**Comment by jonovs** [comment]
> What tools do you use for that? I have tried to have some spellchecking as part of Visual Studio Code, but does not seem to handle much more than basic word checking.

I just used the typo inspector in IntelliJ IDEA. The rest of changes were done as I was going through the docs to explore Unleash in more detail.

**Pull request merged**